formats/policy/frontend/examples/policy_area.json

```diff
@@ -29,6 +29,28 @@
           "key": "government_name",
           "display_as_result_metadata": true,
           "filterable": false
+        },
```

Comment:
I did, but they should have been a separate commit for clarity.

I'm about to update the PR so these are the _only_ changes. Hold on to you :tophat: